R/tmb.R

```diff
@@ -406,7 +425,12 @@ h_mmrm_tmb_assert_opt <- function(tmb_object,
 #' @keywords internal
 h_mmrm_tmb_extract_cov <- function(tmb_report, tmb_data, visit_var) {
   d <- dim(tmb_report$covariance_lower_chol)
-  visit_names <- levels(tmb_data$full_frame[[visit_var]])
+  visits <- tmb_data$full_frame[[visit_var]]
```

Comment:
does this work for the spatial case? I thought then we might have `visit_var` being a non-scalar vector, i.e. more than one element?